modules/combined/tests/poro_mechanics/borehole_lowres.i

```diff
@@ -0,0 +1,733 @@
+# Poroelastic response of a borehole.
+#
+# LOWRES VERSION: this version does not give perfect agreement with the analytical solution
```

Comment:
oops, thanks for noting that @tonkmr.  The highres version does agree with the analytical solution, as you would hope!  I'll do a change in a few hours.

a